src/main/kotlin/lotto/domain/LottoGenerator.kt

```diff
@@ -12,11 +12,12 @@ class LottoGenerator {
         return lottoNumbers.map { LottoNumber(it) }
     }
 
-    fun generateLottos(count: Int): Lottos {
-        val lottos = mutableListOf<LottoNumbers>()
-        repeat(count) {
-            val lottoNumbers = LottoNumbers(generateLottoNumbers())
-            lottos.add(lottoNumbers)
+    fun generateLottos(count: Int, manualLottos: Lottos): Lottos {
+        val lottos = mutableListOf<Lotto>()
+        lottos.addAll(manualLottos.lottoNumbers)
+        repeat(count - manualLottos.lottoNumbers.size) {
```

Comment:
```suggestion
        repeat(count - manualLottos.size()) {
```

manualLottos.size() 로 바로 가지고 오는게 좋지 않을까요?